Cargo.toml

```diff
@@ -13,3 +13,4 @@ path = "src/main.rs"
 [dependencies]
 anyhow = "1.0.41"
 structopt = "0.3.21"
+thiserror ="1.0.26"
```

Comment:
GitHub seems to think a newline is needed. I don't mind.

Reply:
done